src/grape/hooks/auth_field.clj

```diff
@@ -7,60 +7,40 @@
       (let [auth-strategy (:auth-strategy resource)
             auth-field? (= (:type auth-strategy) :field)
             doc-field (:doc-field auth-strategy)
+            additional-permission-doc-fields (or (:additional-permission-doc-fields auth-strategy) [])
```

Comment:
The idea is to provide a list of additional keys for which the auth is able to authorise edition.